workspace/notebook/py_sb_raw_to_std.json

```diff
@@ -158,7 +159,7 @@
 				"source": [
 					"def read_raw_messages(source_path: str) -> DataFrame:\n",
 					"    try: \n",
-					"        df: DataFrame = spark.read.json(source_path)\n",
+					"        df: DataFrame = spark.read.json(source_path, schema = spark_schema)\n",
```

Comment:
@muhammadkc this will force the reading of the raw messages into a dataframe that fits the schema exactly. Does this therefore negate the need for the casting of nulls etc? For appeal-has there was a schema mismatch error related to a tuple.

Reply:
@muhammadkc I think the reload is needed to fully test this. I would reload and then run the raw_to_std without the casting of nulls and see if that makes it simpler.